test/basic.html

```diff
@@ -297,7 +297,7 @@
 
       var json = f.serialize();
       assert.equal(Object.keys(json).length, 2);
-      
```

Comment:
In lieu of testing the feature detection, perhaps you can write a test for the side-effect of the feature detection being `true` or `false`? e.g., `test('does not try to raise validation bubbles on browsers that do not have them', ...);`